streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -897,4 +903,45 @@ public void close() {}
             );
         }
     }
+    
+    @Test
+    public void shouldFeedStoreFromGlobalKTable() {
+        final String topicName = "topic";
+        final String globalStoreName = "globalStore";
+        final String key1 = "k1";
+        final String value1 = "value1";
+        final String key2 = "k2";
+        final String value2 = "value2";
+        //
+        final StreamsBuilder builder = new StreamsBuilder();
+        @SuppressWarnings("unused")
+        final GlobalKTable<String,String> globalTable = builder
+        .globalTable(topicName,  
+            Consumed.with(Serdes.String(), Serdes.String()),
+            Materialized.<String,String,KeyValueStore<Bytes,byte[]>>as(globalStoreName))
+        ;
+        try (TopologyTestDriver testDriver = new TopologyTestDriver(builder.build(), config)) {
+            final KeyValueStore<String,String> globalStore = testDriver.getKeyValueStore(globalStoreName);
+            Assert.assertNotNull(globalStore);
+            Assert.assertNotNull(testDriver.getAllStateStores().get(globalStoreName));
+            // push data to the topology
+            final ConsumerRecordFactory<String,String> crf = new ConsumerRecordFactory<>(new StringSerializer(), new StringSerializer());
```

Comment:
nit: `crf` -- we avoid abbreviations because they make the code much harder to read.